### PR TITLE
Add sofi3d.md

### PR DESCRIPTION
**Pull request by EetuHuuskoCSC**
Also made a note where one should link a new app page. Could you take a look at the note @joonas-somero ?

### CODE REVIEW

File: docs/apps/sofi3d.md [hunk 1]
```diff
@@ -0,0 +1,240 @@
...
+
+`sofi3D`
+
+## Example batch job
+
+SOFI3D creates many auxiliary files, so in this example subdirectories are used to reduce clutter.
+
+```bash
+#!/bin/bash
+#SBATCH --account=<your-project-unix-id>
```
Comment (samumantha) [inline docs/apps/sofi3d.md:36]:
```suggestion
#SBATCH --account=<project>
```
I'd update the mentions of your-project-unix-id to what is used in eg the batch job example page, which is <project>. And add a note that that needs to be changed in the script before running with an hint on the format,which is project_200xxxx

File: docs/apps/sofi3d.md [hunk 2]
```diff
@@ -0,0 +1,240 @@
...
+#SBATCH --time=00:05:00
+#SBATCH --partition=small
+
+# Load the prerequisites for sofi3D
+module load intel-oneapi-compilers-classic/2021.6.0
+module load openmpi/4.1.4
+module load sofi3d
+ 
+# Move to working directory
+cd /scratch/<your-project-unix-id>
```
Comment (samumantha) [inline docs/apps/sofi3d.md:48]:
```suggestion
cd /scratch/<project>/<some_dir>
```

**Review by samumantha — COMMENTED** [review]
2 minor comments, otherwise looks good and also works for users :)

**Review by samumantha — APPROVED** [review]
👍

**Review by ktiits — APPROVED** [review]

**Pull request merged**